decred/decred/dcr/account.py

```diff
@@ -1962,7 +1962,7 @@ def sync(self):
         self.updateStakeStats()
         pool = self.stakePool()
         if pool:
-            pool.authorize(self.votingAddress())
+            pool.authorize(self.votingAddress().string())
```

Comment:
Starting to doubt my rebase was perfect. Started to test some more and couldn't buy tickets. Possibly a problem also in master though.

Reply:
It looks like I can buy, there's just that error looking output afterwards.